From 44b8617a56c35ffc5d9cc83deb8c2e1ca094cf0f Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Mon, 9 Dec 2024 17:25:34 +0800
Subject: [PATCH 237/316] AOSCOS: gpio: gpio-nct6102: add a missing include to
 <linux/gpio/driver.h>

This driver was missing an include to <linux/gpio/driver.h>, the reference
to which was likely forked out at some point after the original driver
code was committed back in 2018.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/gpio/gpio-nct6102.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpio/gpio-nct6102.c b/drivers/gpio/gpio-nct6102.c
index 25efc588c761..547b61008390 100644
--- a/drivers/gpio/gpio-nct6102.c
+++ b/drivers/gpio/gpio-nct6102.c
@@ -6,6 +6,7 @@
 #include <asm/types.h>
 #include <linux/gpio.h>
 #include <linux/io.h>
+#include <linux/gpio/driver.h>
 
 #define NCT6102_GPIO_BASE 	80
 
-- 
2.48.1

